src/components/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import Accordion from './Accordion';
+import AccordionDetail from './AccordionDetail';
+import AccordionHeader from './AccordionHeader';
+
```

Comment:
https://github.com/patricklindev/pat-ui/blob/production/src/components/Dropdown/index.tsx

Combine Accordion Related component together

Reply:
I have done this